pdc/apps/component/migrations/0004_auto_20150708_0746.py

```diff
@@ -14,6 +13,7 @@ def create_release_component_types(apps, schema_editor):
         'iso',
         'composite'
     ]
+    ReleaseComponentType = apps.get_model('component', 'ReleaseComponentType')
```

Comment:
what's the benefit but not importing ReleaseComponentType directly? 
I think this migration file won't be applied in latest database.